src/Microsoft.AspNet.Mvc.Core/Infrastructure/MvcRouteHandler.cs

```diff
@@ -87,7 +87,7 @@ public VirtualPathData GetVirtualPath(VirtualPathContext context)
                 {
                     _logger.ExecutingAction(actionDescriptor);
 
-                    var startTime = Environment.TickCount;
+                    var startTime = DateTime.UtcNow;
```

Comment:
Ideally, a `Stopwatch` would only be allocated after an `IsEnabled(LogLevel.Information)` check, to avoid instantiating it abusively (optionally, I guess `Stopwatch` instances could be pooled). IMHO, profiling should be a pay-to-play scenario.